axon-5/api-changes.md

```diff
@@ -195,6 +200,23 @@ As can be expected, the `MessageStream` streams implementation of `Message`. Hen
 Framework uses this `Context` to add the aggregate identifier, aggregate type, and sequence number for events that
 originate from an aggregate-based event store (thus a pre-Dynamic Consistency Boundary event store).
 
+### MetaData with String values
+
+The `MetaData` class in Axon Framework changed its implementation. Originally, it was a `Map<String, ?>` implementation.
+As of Axon Framework 5, it is a `Map<String, String`.
+
+The reason for this shift can be broken down in two main pillars:
+
+1. It greatly simplifies de-/serialization for storing `Messages` and putting `Messages` over the wire, since any value
+   is a `String` in all cases.
+2. It aligns better with how other services, libraries, and frameworks view metadata, which tends to be a `String` or
```

Comment:
3. Depending on requirements, serialization for different values can be different.

Reply:
Fair addition. Will add it.